sled-agent/src/support_bundle/storage.rs

```diff
@@ -553,6 +595,22 @@ impl<'a> SupportBundleManager<'a> {
             .await?;
         info!(log, "Dataset does exist for bundle");
 
+        // The mounted root of the support bundle dataset
+        let support_bundle_dir = dataset
+            .ensure_mounted_and_get_mountpoint(
```

Comment:
This **technically** is not tested - the call to `dyn_nested_dataset_ensure` above should actually do the mounting, so this call shouldn't actually be necessary. However, I included it here out of consistency with the other two callsites.

(This isn't easy to test; I'd basically need to stop this function from running halfway through. But it seems like a "paranoid check" anyway).